src/helm/benchmark/scenarios/medhallu_scenario.py

```diff
@@ -0,0 +1,103 @@
+from typing import List
+from datasets import load_dataset
+
+from helm.common.hierarchical_logger import hlog
+from helm.benchmark.scenarios.scenario import (
+    Scenario,
+    Instance,
+    Reference,
+    TEST_SPLIT,
+    CORRECT_TAG,
+    PassageQuestionInput,
+    Output,
+)
+
+
+class MedHalluScenario(Scenario):
+    """
+    MedHallu is a medical hallucination dataset that consists of PubMed articles and associated questions,
+    with the objective being to classify whether the answer is factual or hallucinated.
+    MedHallu: https://medhallu.github.io/
+    """
+
+    name = "medhallu"
+    description = "A dataset which consists of a PubMed article and a question "
```

Comment:
nit: "A dataset of PubMed articles and associated questions, with the objective being to classify whether the answer is factual or hallucinated."